usaspending_api/etl/management/commands/es_rapidloader.py

```diff
@@ -264,13 +188,24 @@ def process_cli_parameters(options: dict, es_client) -> None:
 
 def set_config(copy_args: list, arg_parse_options: dict) -> dict:
     """Set values based on env vars and when the script started"""
-    config = {
-        "aws_region": settings.USASPENDING_AWS_REGION,
-        "s3_bucket": settings.DELETED_TRANSACTIONS_S3_BUCKET_NAME,
-        "root_index": settings.ES_TRANSACTIONS_QUERY_ALIAS_PREFIX,
-        "processing_start_datetime": datetime.now(timezone.utc),
-        "verbose": arg_parse_options["verbosity"] > 1,  # convert the management command's levels of verbosity to a bool
-    }
+    if arg_parse_options["type"] == "transactions":
+        config = {
```

Comment:
Most of config is the same. Either move the only difference (`root_index`)  into a variable before definining config, or set root_index after config is created

root_index = settings.ES_AWARDS_QUERY_ALIAS_PREFIX, if arg_parse_options["type"] == "awards" else settings.ES_TRANSACTIONS_QUERY_ALIAS_PREFIX

config = {
...
"root_index": root_index
...
}



OR

config = {....}

config["root_index"] = settings.ES_AWARDS_QUERY_ALIAS_PREFIX, if arg_parse_options["type"] == "awards" else settings.ES_TRANSACTIONS_QUERY_ALIAS_PREFIX